packages/client/src/Conversation.tsx

```diff
@@ -212,17 +261,27 @@ function PresentableEdited({
     presentable.editedBy.mailbox !== presentable.from.mailbox
   return (
     <span className={classes.edited}>
-      Edited {moment(presentable.editedAt).calendar()}
-      {editorIsntAuthor ? (
-        <>
-          {" "}
-          by <Participant {...presentable.editedBy} />
-        </>
-      ) : null}
+      {displayPresentableEdited(presentable)}
     </span>
   )
 }
 
+function displayPresentableEdited({
+  editedAt,
+  editedBy,
+  from
+}: graphql.Presentable) {
+  //const classes = useStyles()
+  if (!editedAt || !editedBy) {
+    return null
+  }
+  const editorIsntAuthor =
+    editedBy.host !== from.host || editedBy.mailbox !== from.mailbox
+  return `Edited ${moment(editedAt).calendar()} ${
+    editorIsntAuthor ? ` by ${<Participant {...editedBy} />}` : ""
```

Comment:
This is nice.

Reply:
Oh, I overlooked something: this function returns strings, so you cannot call the `Participant` react component. When I view an edited conversation with this code this is what I see:

> Last Friday at 11:52 AM Edited Last Friday at 12:00 PM by [object Object]

React elements and strings do not mix. The only thing you can put inside a string is another string.